r2t2/core.py

```diff
@@ -78,7 +78,12 @@ def add_reference(
     @wrapt.decorator(enabled=lambda: BIBLIOGRAPHY.track_references)
     def wrapper(wrapped, instance, args, kwargs):
         source = inspect.getsourcefile(wrapped)
-        line = inspect.getsourcelines(wrapped)[1]
+        try:
+            line = inspect.getsourcelines(wrapped)[1]
+        # add exception for testing notebook in a subprocess
+        except OSError:
+            line = "n/a"
```

Comment:
Should this be more informative?

Reply:
it only appears when testing the notebook, in a subprocess - I don't think a user would ever see it

Reply:
Okay.